### DISCUSSION TITLE
Implement a gluon mode

### DISCUSSION THREAD

**Discussion by tiziodcaio** [filips123/PWAsForFirefox#438] [Ideas]
I've seen this project and I thought it might be interesting for future developing...

https://github.com/gluon-framework/gluon

**Comment by filips123** [comment]
Well, what exactly should this Gluon mode do? It seems like Gluon already supports normal Firefox.

**Comment by tiziodcaio** [comment]
It is an idea I had. Because the gluon project could implement a pseudo-pwa using the original firefox without patching it in the system folder but only in the profile. I thought it would be possible to implement this for PWAsForFirefox too, to avoid downloading twice firefox.

**Reply by filips123** [reply]
Gluon only changes preferences and CSS for Firefox, which is possible just by modifying the profile. However, PWAsForFirefox also needs to modify internal Firefox JS, which is AFAIK only possible by adding a loader to the runtime/system directory.
